### PR TITLE
Included Sandra's suggested changes. Kept mailto only for Terry's email.

### PR DESCRIPTION
**Pull request by dianam21**

### CODE REVIEW

File: code/Phase_1a_Report.Rmd [hunk 1]
```diff
@@ -1158,14 +1158,15 @@ Tel: 0131 275 7481
...
-Data Driven Innovation, Public Health Scotland  
-Tel: 0131 275 6722  
-Email: [srobb@nhs.net](srobb@nhs.net)
-
 **Diana Marosi**  
 _Senior Information Analyst_  
 Data Driven Innovation, Public Health Scotland  
 Tel: 0131 275 1073  
-Email: [diana.marosi@nhs.net](mailto:diana.marosi@nhs.net) 
+Email: [diana.marosi@nhs.net](diana.marosi@nhs.net)
```
Comment (terrymclaughlin) [inline code/Phase_1a_Report.Rmd:1165]:
```suggestion
Email: [diana.marosi@nhs.net](mailto:diana.marosi@nhs.net) 
```

File: code/Phase_1a_Report.Rmd [hunk 2]
```diff
@@ -1158,14 +1158,15 @@ Tel: 0131 275 7481
...
 Tel: 0131 275 1073  
-Email: [diana.marosi@nhs.net](mailto:diana.marosi@nhs.net) 
+Email: [diana.marosi@nhs.net](diana.marosi@nhs.net) 
+
+
+**Sandra Robb**  
+_Information Consultant_  
+Data Driven Innovation, Public Health Scotland  
+Tel: 0131 275 6722  
+Email: [srobb@nhs.net](srobb@nhs.net)
```
Comment (terrymclaughlin) [inline code/Phase_1a_Report.Rmd:1172]:
```suggestion
Email: [srobb@nhs.net](mailto:srobb@nhs.net)
```

**Comment by dianam21** [comment]
Thank you, Terry! I just assumed that because Sandra didn't have a mailto, you were the only one to have the 'mailto'. I had a 50-50 change and I chose incorrectly :). I hope by committing one change, th other will to, as I can't select it (for sandra's email). Let's see.

**Comment by terrymclaughlin** [comment]
Thanks, Diana!

**Pull request merged**